src/widgets/partnered/ui/partnered.module.scss

```diff
@@ -0,0 +1,41 @@
+.partnered-content {
+  display: flex;
+  flex-direction: column;
+
+  .title {
+    font-size: 36px;
+    font-weight: $font-weight-medium;
+    line-height: 44px;
+    color: $color-black;
+    letter-spacing: 0;
+  }
+
+  .partners {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    align-self: center;
+    justify-content: space-between;
+
+    width: 100%;
+    max-width: 500px;
+    padding: 16px;
+
+    .partner-logo-container {
+      display: flex;
+      flex-basis: 20%;
+      align-items: center;
+      justify-content: center;
+
+      @include media-mobile {
+        flex-basis: 50%;
+        padding: 24px 0;
+      }
+    }
+
+    @include media-tablet {
+      row-gap: 16px;
+      padding: 16px 0;
+    }
```

Comment:
In the project we try to avoid using paddings and margin if possible. Perhaps this property could be removed here as well

Reply:
Replaced them with gaps